### PR TITLE
Allow string features use same family name without adding prefix for …

### PR DESCRIPTION
**Pull request by jq**
…each feature.

@deerzq @bhntr @spencerde

### CODE REVIEW

File: core/src/main/java/com/airbnb/aerosolve/core/features/Features.java [hunk 1]
```diff
@@ -4,16 +4,18 @@
...
-import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.lang3.tuple.ImmutablePair;
 import org.apache.commons.lang3.tuple.Pair;
 
 import java.lang.reflect.Field;
 import java.util.*;
 
-@Builder @Slf4j
 public class Features {
+  public final static String DEFAULT_STRING_FAMILY = "L";
```
Comment (deerzq) [inline core/src/main/java/com/airbnb/aerosolve/core/features/Features.java]: I feel we need a more distinguishable name, what if there are other existing feature family called 'L'?

**Comment by deerzq** [comment]
I feel you could just add a prefix "L_" to all string features in our pricing table to avoid having the complication here.

File: core/src/main/java/com/airbnb/aerosolve/core/features/Features.java (hunk 1 above)
Comment (jq) [inline core/src/main/java/com/airbnb/aerosolve/core/features/Features.java]: in this case, can use genFeaturesWithCustomerStringFamily

**Comment by jq** [comment]
@deerzq add L_ to all string features will require online scoring to update all the naming too. so add it here to reduce the work on online scoring.

**Comment by deerzq** [comment]
I see.

File: core/src/main/java/com/airbnb/aerosolve/core/features/Features.java [hunk 2]
```diff
@@ -25,6 +27,39 @@
 
   public final String[] names;
   public final Object[] values;
+  private final String defaultStringFamilyName;
+
+  public static Features genDefaultStringFamilyFeatures(String[] names, Object[] values) {
+    return new Features(names, values);
+  }
+
+  public static Features genFeaturesWithCustomerStringFamily(
```
Comment (spencerde) [inline core/src/main/java/com/airbnb/aerosolve/core/features/Features.java]: sp/Customer/Custom

File: core/src/main/java/com/airbnb/aerosolve/core/features/Features.java (hunk 1 above)
Comment (spencerde) [inline core/src/main/java/com/airbnb/aerosolve/core/features/Features.java]: But could this have unintended consequences if people didn't realize this was 'L' and had another family of the same name coincidentally?

**Pull request closed without merging**